src/uninstall.js

```diff
@@ -82,9 +61,6 @@ Delete the installed package(s) from the ~/.pulsar/packages directory.\
             if (fs.existsSync(packageManifestPath)) {
               const packageVersion = this.getPackageVersion(packageDirectory);
```

Comment:
Whoops, didn't notice `packageVersion` is now defined but unused.

Didn't notice I could/should delete this line, too. Can do it in a follow-up, I guess.